code/_onclick/hud/ability_screen_objects.dm

```diff
@@ -1,4 +1,56 @@
-/obj/screen/movable/ability_master
+/// Encode a mob's client view size into a screen_loc X component.
+/proc/encode_screen_X(var/X, var/mob/viewer)
```

Comment:
Sorry but I hate the casing. The `x` is not capitalized even at dm-level in general, and the next line calls a proc with x not capitalized. I mean specifically the proc name, not the arg, and similarly for the other procs below here.

Reply:
Casing fixed.